apps/i18n/common/en_us.json

```diff
@@ -1867,27 +1867,25 @@
   "myProfessionalLearningCourses": "My Professional Learning Courses",
   "myWorkshops": "My Workshops",
 
-  "accountInformationAccountInformation": "Account information",
-  "accountInformationVerifiedTeacher": "Verified Teacher",
-  "accountInformationDisplayName": "Display Name",
-  "accountInformationUsername": "Username",
-  "accountInformationDisplayNameHintTeacher": "This is what your students see.",
-  "accountInformationEmail": "Email address",
-  "accountInformationEmailHint": "Make sure you can receive emails to this email address.",
-  "accountInformationPassword": "Password",
-  "accountInformationPasswordConfirmation": "Password confirmation",
-  "accountInformationCurrentPassword": "Confirm current password",
-  "accountInformationCurrentPasswordHint": "We need your current password to confirm changes.",
-  "accountInformationUpdateAccountInformation": "Update account information",
-  "accountInformationNoPasswordBecauseSponsored": "See your teacher to manage your password.",
-  "accountInformationGender": "Gender (optional)",
-  "accountInformationState": "State",
-  "accountInformationUpdateFieldParentPermissionRequired": "(You must obtain parental permission before updating this field)",
-  "accountInformationUpdateEmail": "Update email",
-  "accountInformationSelectState": "Select state",
-  "accountInformationReviewErrors": "Review errors above and try again.",
-  "accountInformationUpdateSuccess": "Account information successfully updated.",
-  "accountInformationEmailUpdateSuccess": "Email successfully updated.",
+  "accountInformation_accountInformation": "Account information",
+  "accountInformation_verifiedTeacher": "Verified Teacher",
+  "accountInformation_displayName": "Display Name",
+  "accountInformation_username": "Username",
```

Comment:
Username is a generic enough string that we could reuse in other components. I would add the string `"username": "Username"`, instead of `"accountInformation_username": "Username"`
```suggestion
```